prisma/migrations/20220715134600_init/migration.sql

```diff
@@ -100,6 +100,43 @@ CREATE TABLE `WalletsOnUserProfile` (
     PRIMARY KEY (`walletId`, `userProfileId`)
 ) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 
+-- CreateTable
+CREATE TABLE `Price` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `value` DECIMAL(64, 16) NOT NULL,
```

Comment:
Probably could get away with 8 decimals as I doubt we're going to be touching BTC at all.